jta-crash-rec-eap7/src/main/webapp/WEB-INF/jta-crash-rec-quickstart-ds.xml

```diff
@@ -24,7 +24,7 @@
     xsi:schemaLocation="http://www.jboss.org/ironjacamar/schema http://docs.jboss.org/ironjacamar/schema/datasources_1_0.xsd">
     <xa-datasource jndi-name="java:jboss/datasources/JTACrashRecQuickstartDS" pool-name="JTACrashRecQuickstart" enabled="true">
         <xa-datasource-property name="URL">
-            jdbc:h2:file:${jboss.server.data.dir}/jta-crash-rec-quickstart;DB_CLOSE_ON_EXIT=FALSE;DB_CLOSE_DELAY=-1
+            jdbc:h2:file:${jboss.server.data.dir}/../../jta-crash-rec-quickstart;DB_CLOSE_ON_EXIT=FALSE;DB_CLOSE_DELAY=-1
```

Comment:
This seems a little flimsy, but okay.  (Maybe something ${env.JBOSS_HOME}/standalone/data might be a bit safer.)

Reply:
@rcernich agree, do you think I should create a new PR with the change you point out?

Reply:
I don't think it's critical.  We can create a new issue if we think it's worth our time addressing.  I made the comment mostly for future reference.